services/QuillLMS/spec/controllers/api/v1/active_activity_session_controller_spec.rb

```diff
@@ -341,7 +338,7 @@
         end
       end
     end
-    # rubocop:enable RSpec/Pending, RSpec/ExampleLength
+    # rubocop:enable RSpec/ExampleLength
```

Comment:
I vote to remove this cop, rather than add exceptions. In general, the pros of relaxed linting in testland outweigh the cons.

Reply:
I think I generally agree with the sentiment.  Might be worth excluding our `spec` directories from this cop.

Reply:
Which cop?

Reply:
Are we talking about RSpec/ExampleLength or RSpec/Pending?